platform_versioned_docs/version-23.4.0/data/data-explorer.mdx

```diff
@@ -88,6 +88,22 @@ Data Explorer supports file uploads to your private cloud storage buckets. From
 You must configure cross-origin resource sharing (CORS) for your cloud storage provider to allow file uploads from Seqera. CORS configuration differs for each provider.
 :::
 
+### Download multiple files
+
+You can download multiple files via the user interface or using a script in your cloud provider's CLI. 
+
+:::note
```

Comment:
I don't think this is correct. Downloading multiple folders/files works, but the folder structure is not preserved in non-Chromium based browsers.

Reply:
@robnewman 

> FF/Safari download of folders doesn't preserve the folder-file structure but instead concatenates the folder hierarchy and file name. This is an acceptable work around for now.

What does the concatenated filename look like? Thanks!